manta-pay/src/crypto/ecc/arkworks.rs

```diff
@@ -270,6 +274,50 @@ where
     }
 }
 
+impl<C> ConditionalSelect for Group<C>
+where
+    C: ProjectiveCurve,
+{
+    #[inline]
+    fn select(bit: &Bool<()>, true_value: &Self, false_value: &Self, compiler: &mut ()) -> Self {
+        let _ = compiler;
+        if *bit {
+            *true_value
+        } else {
+            *false_value
+        }
+    }
+}
+
+impl<C> cmp::PartialEq<Self> for Group<C>
+where
+    C: ProjectiveCurve,
+{
+    #[inline]
+    fn eq(&self, rhs: &Self, compiler: &mut ()) -> Bool<()> {
+        let _ = compiler;
+        self == rhs
+    }
+}
+
+impl<C> Zero for Group<C>
+where
+    C: ProjectiveCurve,
+{
+    type Verification = bool;
+
+    #[inline]
+    fn zero(compiler: &mut ()) -> Self {
+        let _ = compiler;
+        Group(<C::Affine as ark_std::Zero>::zero())
```

Comment:
If this doesn't work, then use `ark_ff::Zero` instead, but don't use `ark_std` in general if you can (it's a cursed crate).

```suggestion
        Self(C::Affine::zero())
```